docs/application/native/guides/ui/efl/efl-util.md

```diff
@@ -375,6 +387,97 @@ To generate touch input events:
    }
    ```
 
+## Generating Point Input Events
+
+To generate point input events, follow these steps:
+
+1. Create the `efl_util_inputgen_h` structure and initialize the structure members with `efl_util_input_initialize_generator()`, `efl_util_input_initialize_generator_with_name()` or `efl_util_input_initialize_generator_with_sync()`:
```

Comment:
Please add a comma (,) before "or".
It is recommended to use the serial comma rule.